coordinator/gscoordinator/utils.py

```diff
@@ -687,6 +683,137 @@ def _tranform_dataframe_selector(context_type, schema, selector):
     return json.dumps(selector)
 
 
+def _transform_vertex_data_v(selector):
+    if selector not in ("v.id", "v.data"):
+        raise SyntaxError("selector of v must be 'id' or 'data'")
+    return selector
+
+
+def _transform_vertex_data_e(selector):
+    if selector not in ("e.src", "e.dst", "e.data"):
+        raise SyntaxError("selector of e must be 'src', 'dst' or 'data'")
+    return selector
+
+
+def _transform_vertex_data_r(selector):
+    if selector != "r":
+        raise SyntaxError("selector of r must be 'r'")
+    return selector
+
+
+def _transform_vertex_property_data_r(selector):
+    # The second part of selector or r is user defined name.
+    # So we will allow any str
+    return selector
+
+
+def _transform_labeled_vertex_data_v(schema, label, prop):
+    label_id = schema.get_vertex_label_id(label)
+    if prop == "id":
+        return "label{}.{}".format(label_id, prop)
+    else:
+        prop_id = schema.get_vertex_property_id(label, prop)
+        return "label{}.property{}".format(label_id, prop_id)
+
+
+def _transform_labeled_vertex_data_e(schema, label, prop):
+    label_id = schema.get_edge_label_id(label)
+    if prop in ("src", "dst"):
+        return "label{}.{}".format(label_id, prop)
+    else:
+        prop_id = schema.get_vertex_property_id(label, prop)
+        return "label{}.property{}".format(label_id, prop_id)
+
+
+def _transform_labeled_vertex_data_r(schema, label):
+    label_id = schema.get_vertex_label_id(label)
+    return "label{}".format(label_id)
+
+
+def _transform_labeled_vertex_property_data_r(schema, label, prop):
+    label_id = schema.get_vertex_label_id(label)
+    return "label{}.{}".format(label_id, prop)
+
+
+def transform_vertex_data_selector(selector):
```

Comment:
Give some comments like `transform_labeled_vertex_data_selector` to give user a high level view of how selector works. The same applies to `transform_vertex_property_data_selector`.